tssc/step_implementers/sign_container_image/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""tssc.StepImplementers for the 'sign-container-image' TSSC step.
+
+
+Step Configuration
+------------------
+All tssc.StepImplementers for this step should
+accept minimally the following configuration options.
+
+| Configuration Key                           | Required? | Default | Description
+|---------------------------------------------|-----------|---------|-------------
+| `container-image-signature-server-url`      | True      |         | Url where the signature /
+                                                                      server is located
+| `container-image-signature-server-username` | True      |         | Username to log onto /
+                                                                      the signature server
+| `container-image-signature-server-password` | True      |         | Password to log onto /
+                                                                      the signature server
+| `container-image-signer-pgp-private-key`    | True      |         | PGP Private Key /
+                                                                      used to sign
+
+
+Results
+-------
+All tssc.StepImplementers for this step should
+minimally produce the following step results.
+
+| Result Key                                          | Description
+|-----------------------------------------------------|------------
+| `container-image-signature-private-key-fingerprint` | Fingerprint for the private key for /
+                                                        image signing
+| `container-image-signature-file-path`               | File path where signature is located /
+                                                        eg) /tmp/jkeam/hello-node@/
```

Comment:
I think we should change the example reference paths (here and below) from `jkeam` to a non-developer referenced path such as 'tssc-developer'. This will improve clarity for new community members who do not know what a Jon Keam is.

Reply:
done